CMakeLists.txt

```diff
@@ -149,7 +149,9 @@ set(IGNITION_GAZEBO_GUI_PLUGIN_INSTALL_DIR
 #============================================================================
 # Configure the build
 #============================================================================
-ign_configure_build(QUIT_IF_BUILD_ERRORS)
+ign_configure_build(QUIT_IF_BUILD_ERRORS
+  COMPONENTS model
```

Comment:
> it was separated into a new folder following the log structure

Usually following an existing pattern is a good idea, but I'm not sure that creating a separate component is necessary in this case.

On `ign-transport` the log feature was kept in a separate component because it brings a new dependency which we didn't want to add to the core of the library (SQLite). But in this case, the `model` command is not bringing any new dependencies and I think it's not sufficiently separate from the core to justify being its own component. 

Would it be possible to move the new command into `src/cmd`? I think this should also simplify a lot of the current CMake code.

Reply:
I made a [commit](https://github.com/ignitionrobotics/ign-gazebo/pull/893/commits/2f8d28cb248dcab948feeeb1bf2d25b2aee0cda7) with only this change so you can have a look at it. I didn't put [cmdgazebo.rb.in](https://github.com/ignitionrobotics/ign-gazebo/blob/wagnermarcos/CLI_model_info_dump/src/cmd/cmdgazebo.rb.in) and [cmdmodel.rb.in](https://github.com/ignitionrobotics/ign-gazebo/blob/wagnermarcos/CLI_model_info_dump/src/cmd/cmdmodel.rb.in) together because the gazebo command implements many defaults value and sub-commands itself and mixing them would be difficult to follow. Because of this I had to also leave [gazebo.yaml.in](https://github.com/ignitionrobotics/ign-gazebo/blob/wagnermarcos/CLI_model_info_dump/src/cmd/gazebo.yaml.in) and [ignmodel.yaml.in](https://github.com/ignitionrobotics/ign-gazebo/blob/wagnermarcos/CLI_model_info_dump/src/cmd/ignmodel.yaml.in) in different files so the command find different ruby files to process it. 

Putting it all together was my initial approach when I first started so its doable, but I thought it was really messy and I had to adapt some of the gazebo command implementation so it could coexist with the model one, so thats the reason I decided to look for a way to keep them in different files.

Let me know what you think about it.

Reply:
See #928 for a possible solution